app/views/events/show.html.erb

```diff
@@ -16,9 +16,10 @@
       </div>
       <div class="panel-body">
         <% if @event.owner %>
-          <%= link_to(url_for_twitter(@event.owner)) do %>
-            <%= image_tag @event.owner.image_url %>
-            <%= "@#{@event.owner.nickname}" %>
+          <% if @event.owner.provider == 'twitter' %>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)